lib/node_modules/@stdlib/blas/ext/base/sapxsumkbn2/manifest.json

```diff
@@ -34,7 +34,12 @@
 				"-lm"
```

Comment:
Need to change to 2-space indentation and update to have configurations for benchmarks and examples.